iianmelo/citigames-squadian
# feat/component/dialogEnterGame

What I did
- Created the Dialog Enter Game Menu
- Linked the Enter Button and the Login Menu

How to test
	1	Open the terminal and go to the folder of this project
	2	In terminal:
	i	git checkout main
	ii	git pull
	iii	git checkout <feat/component/dialogEnterGame>
	iv	git pull origin <feat/component/dialogEnterGame>
	v	yarn install
	vii	yarn dev
	3	In your browser go to http://localhost:3000
	4	Check if the component looks just like the mockup in different screen sizes
	5	In VScode open <relative path>
	6	Things to change in the code during the test
if (isAwesome){
	7	  return <Component />
	8	}

	9	Read the changed files section on GitHub
	10	Test the things that you think are worth testing, even the ones that are not in this description
